tasks/vm_state_present.yml

```diff
@@ -0,0 +1,124 @@
+---
+- block:
+  - name: Create VMs
+    include: create_vms.yml
+    with_items: "{{ vms }}"
```

Comment:
You can use in this tasks file `create_vms` isntead `vms`, so you don't need to rewrite it in `main.yml`